src/components/LogIn.js

```diff
@@ -0,0 +1,93 @@
+import React, { Fragment, Component } from 'react';
+import TextField from '@material-ui/core/TextField';
+import Button from '@material-ui/core/Button';
+import { logIn } from '../services/userService';
+import { Redirect } from 'react-router-dom';
+import '../App.css';
+
+
+export default class Form extends Component {
+  state = {
+    login: '',
+    password: '',
+    notification: '',
+    notificationType: '',
+    redirect: false
+  }
+
+  handleChangeInput = e => {
+    this.setState({[e.target.name]: e.target.value});
+  }
+
+  handleSubmit = e => {
+    e.preventDefault();
+    const { login, password } = this.state;
+    logIn({ login, password })
+                .then(result => {
+                  // this.showNotification(result, 'notification success');
+                  // this.setState({notification: result, notificationType: 'notification success' });
+                  if(result) {
+                    this.setState({redirect: true});
+                  }
+                })
+                .catch(error => {
+                  this.showNotification(error, 'notification error');
+                  // this.setState({notification: error, notificationType: 'notification error' });
+                });
+
+    // setTimeout(this.hideNotification, 5000);
+
+  }
+
+  showNotification = (notification, notificationType, time = 5000) => {
+    this.setState({notification: notification, notificationType: notificationType});
```

Comment:
this.setState({notification: notification, notificationType: notificationType}); Can be simplified to this.setState({ notification, notificationType })

Reply:
done